test/testrunner.py

```diff
@@ -116,7 +128,7 @@ def main():
     specific_test_type = re.sub("benchmark-", "", test_type) if benchmark_mode else test_type
     test_path = os.path.join("benchmark", specific_test_type) if benchmark_mode else specific_test_type
 
-    if specific_test_type in ("sanity", "ecs", "ec2", "eks", "canary"):
+    if specific_test_type in ("sanity", "ecs", "ec2", "eks", "canary", "bai"):
         report = os.path.join(os.getcwd(), "test", f"{test_type}.xml")
```

Comment:
We need to set the test_path for "bai" because the scripts are located in "benchmark/bai" folder, otherwise it would not be found by pytest